Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -30,27 +33,64 @@ @interface FSCalendarStickyHeader (Private)
 
 @end
 
+@interface BPKCalendarStickyHeader ()
+
+@property(weak, nonatomic, readonly) BPKButton *selectMonthButton;
+
+@end
+
 @implementation BPKCalendarStickyHeader
 
+CGFloat const BPKBaselineOffset = -5.0;
```

Comment:
Could this use a BPKSpacing token? I think BPKSpacingSm (4) or BPKSpacingMd (8) would fit?

Reply:
Hmm ok, just a tiny bit worried what will happen if we go through another font alignment and font-sizes change etc.
We can keep this as is for now then.